fiat-roles/src/main/groovy/com/netflix/spinnaker/fiat/permissions/DefaultPermissionsResolver.java

```diff
@@ -57,27 +57,52 @@
   @Setter
   private ServiceAccountProvider serviceAccountProvider;
 
+  @Value("${auth.anonymous.enabled}")
+  @Setter
+  private boolean anonymousEnabled;
+
+  @Override
+  public Optional<UserPermission> resolveAnonymous() {
+    if (!anonymousEnabled) {
+      return Optional.empty();
+    }
+
+    val groups = new ArrayList<String>(0);
+    val accounts = accountProvider.getAccounts(groups);
```

Comment:
No - it's hitting clouddriver every time. I'll definitely have to address the retry issue.